VTUL/VIVO_VT_Theme
# Update VIVO home Statistics links

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1637) (:star:)

# What does this Pull Request do? (:star:)
Update links in the Statistics section on the VIVO homepage 

Point each icon to the correct URL: 
1. People to https://vivo-dev.cloud.lib.vt.edu/people 
2. Organizations to https://vivo-dev.cloud.lib.vt.edu/organizations 
3. Events to https://vivo-dev.cloud.lib.vt.edu/events
4. Research to https://vivo-dev.cloud.lib.vt.edu/research

Also update the steps in the README.MD file

Also change the structure of this repo

# What's the changes? (:star:)
This PR update a freemarker lib file: lib-home-page.ftl

# How should this be tested?
Please follow the steps in the README.MD 
After apply the vtech themes, go to the VIVO homepage, you will see there are only 4 icon in the statistics section. Each icon now will redirect to the correct URL. 
e.g. 
 People to https://vivo-dev.cloud.lib.vt.edu/people, etc

# Interested parties
Tag (@VTUL/dld-dev ) interested parties

<img width="1045" alt="screen shot 2018-11-12 at 2 28 50 pm" src="https://user-images.githubusercontent.com/7999195/48370434-51136880-e687-11e8-9411-4a66f4bb97af.png">
